website/snippets/_cloud-environments-info.md

```diff
@@ -28,11 +28,33 @@ Both development and deployment environments have a section called **General Set
 | Default to Custom Branch | ☑️ | Determines whether to use a branch other than the repository’s default  | See below |
 | Custom Branch | dev | Custom Branch name | See below |
 
-:::note About dbt version
+### About versions in dbt Cloud
 
-- dbt Cloud allows users to select any dbt release. At this time, **environments must use a dbt version greater than or equal to v1.0.0;** [lower versions are no longer supported](/docs/dbt-versions/upgrade-core-in-cloud).
-- If you select a current version with `(latest)` in the name, your environment will automatically install the latest stable version of the minor version selected.
+- Multiple dbt releases: dbt Cloud enables users to choose from various dbt versions.
+- Version requirements: Environments must use dbt version v1.0.0 or higher. [Lower versions are not supported](/docs/dbt-versions/upgrade-core-in-cloud).
+- Automatic updates: Selecting a version labeled `(latest)` ensures your environment automatically installs the latest stable release of that minor version selected.
+
+
+### Git repository caching 
+:::info Available for dbt Cloud Enterprise plans
+This feature is exclusively available for dbt Cloud Enterprise plans. [Book a demo](https://www.getdbt.com/contact) to learn more.
 :::
```

Comment:
sugg moving this callout to the end of the section. i've been taught that a callout after a title is not "good design" practice

Reply:
We do this in numerous places. I thought the idea was to give the important info at the top of the article, rather than at the tail end, especially if the feature is not available to a subset of users.

@runleonarun  and @mirnawong1 thoughts on this?

Reply:
this is a great question and all great points! i would prefer having the callout out at the top of the content:
- we do this already in pretty much all our docs so adding it at the end would be more of an antipattern. 
- i think having at the top sets clear expectations and also hopefully perks interests.
- when i first read this, i thought this was for all plans and then was surprised it was for enterprise. i almost missed the callout, in fact. so i do prefer it to be at the top.

Reply:
yeah, we have prior art for both. 

i like to talk about the cool thing first instead of dropping caveats/limitations at the top. not a big deal tho. non-blocker for me.